.github/workflows/documentation-validation-workflow.yml

```diff
@@ -0,0 +1,44 @@
+name: Build Docs
+
+on:
+  push:
```

Comment:
really useful ? only on PR looks alright to me

Reply:
correct!